actions/workflows/bwc_finalize_release.yaml

```diff
@@ -43,26 +43,9 @@ tasks:
   finalize:
     next:
       - do:
-          - make_ewc_docs_stable
-  make_ewc_docs_stable:
```

Comment:
EDIT: It turns out we don't actually push any commits to this repo as part of the release process so this step is still broken and we need to find a fix for it.

Probably best way to handle that is to push a commit to master and version branch as part of the release process (same as we do with st2docs repo).

Reply:
When we push to st2docs repo, that triggers build for both st2docs and ewcdocs. So we probably don’t need this step at all.

Reply:
Yep, but we still need ipfabrics one (which is broken atm), right?

Reply:
Ipfabrics-docs repo is the EWC docs. They are the same thing.

Reply:
Nvm, you are right - https://github.com/StackStorm/st2docs/blob/master/Makefile#L103

We should indeed be fine and no additional work is needed for ipfabrics-docs repo.